docs/modules/ROOT/pages/how-to-guides/FAQ.adoc

```diff
@@ -34,6 +34,11 @@ At the moment, multiple users would need to access the environment using the sam
 == How does Payara Cloud handle high availability?
 Payara Cloud ensures high availability by automatically distributing your application instances across multiple availability zones. In the event of a failure, it can quickly recover your application on another healthy instance, minimizing downtime.
 
+== Where Does Payara Cloud Host Its Data?
+Your Payara Cloud data, including application binaries and user information, is hosted within Microsoft Azure's EU region.
```

Comment:
Shouldn't be it saying "is hosted within Microsoft Azure's region which is selected during namespace creation by user" ?

Reply:
Good catch. Wording amended.